R/utils.R

```diff
@@ -216,3 +216,68 @@ download_excel <- function(url, archive, verbose = FALSE,
   dt <- as.data.frame(dt)
   return(dt)
 }
+
+
+#' Create github action for a given source
+#' @description Makes a github workflow yaml file for a given source to be used
+#' as an action to check the data as a github action.
+#' @param source character_array The name of the class to create the workflow
+#' for.
+#' @param workflow_path character_array The path to where the workflow file
+#' should be saved. Defaults to '.github/workflows/'
+make_github_workflow <- function(source,
+                                 workflow_path = paste0(
+                                   ".github/workflows/", source, ".yaml"
+                                 )) {
+  template_path <- "inst/github_workflow_template.yaml"
```

Comment:
needs to load from the package when installed (`system.file("github_workflow_template.yaml", package = "covidregionaldata")